manager/apis/motion/v1alpha1/batchtransfer_webhook.go

```diff
@@ -24,9 +24,17 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/webhook"
 )
 
-func (r *BatchTransfer) SetupWebhookWithManager(mgr ctrl.Manager) error {
+const (
+	DefaultFailedJobHistoryLimit     = 5
+	MaxFailedJobHistoryLimit         = 20
+	DefaultSuccessfulJobHistoryLimit = 5
+	MaxSuccessfulJobHistoryLimit     = 20
+	batchTransferImage               = "ghcr.io/fybrik/mover:latest"
+)
+
+func (batchTransfer *BatchTransfer) SetupWebhookWithManager(mgr ctrl.Manager) error {
```

Comment:
why did you rename `r` to batchTransfer is it was asked by one of the linters ?

Reply:
Yes, "stylecheck" raised it.
`ST1016: methods on the same type should have the same receiver name`